### PR TITLE
fix: Remove code with side effect in filter select UI

### PR DESCRIPTION
**Pull request by ryum91**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
Key is being passed to select filter UI using id.
I changed it because side effect may occur.
In addition, option click extended the range to label.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by jung-han** [comment]
Can one of the admins verify this patch?

File: packages/toast-ui.grid/src/css/grid.css [hunk 1]
```diff
@@ -693,13 +694,18 @@
...
+}
+
 .tui-grid-filter-comparator span {
   font-size: 12px;
   color: #333;
   vertical-align: middle;
 }
 
-.tui-grid-filter-comparator input[type='radio'] + label {
+.tui-grid-filter-comparator label:before {
```
Comment (js87zz) [inline packages/toast-ui.grid/src/css/grid.css]: 동작의 차이는 없겠지만 이 경우는 `pseudo-class` 가 아닌 `pseudo-element`로 사용되었으니 `::before`가 용도에 더 적합할 것 같습니다~!

File: packages/toast-ui.grid/src/view/filterOperator.tsx [hunk 2]
```diff
@@ -26,30 +26,27 @@ class FilterOperatorComp extends Component<Props> {
...
-            id="OR"
-            name="filterOperator"
-            value="OR"
-            checked={operator === 'OR'}
-            onChange={this.handleChangeOperator}
-          />
-          <label for="OR" />
-          <span>OR</span>
-        </div>
+        {['AND', 'OR'].map(value => {
```
Comment (js87zz) [inline packages/toast-ui.grid/src/view/filterOperator.tsx]: `value` 대신 `operatorType` 이 더 명확하고 좋을 것 같습니다~!

File: packages/toast-ui.grid/src/view/selectFilter.tsx [hunk 3]
```diff
@@ -88,9 +87,14 @@ class SelectFilterComp extends Component<Props> {
                 className={cls('filter-list-item', [checked, 'filter-list-item-checked'])}
                 key={text}
               >
-                <input type="checkbox" id={text} checked={checked} onChange={this.handleChange} />
-                <label for={text} />
-                <span>{value}</span>
+                <label>
```
Comment (js87zz) [inline packages/toast-ui.grid/src/view/selectFilter.tsx]: `label` 클릭 범위가 행(`li`) 전체로 확대되면 더 사용성에 좋을 것 같은대..관련 css 작업도 해주실 수 있을까요?

File: packages/toast-ui.grid/src/css/grid.css (hunk 1 above)
Comment (ryum91) [reply packages/toast-ui.grid/src/css/grid.css]: 그런 차이점이 있는 줄 몰랐네요. 감사합니다 !

**Comment by js87zz** [comment]
ok to test

**Pull request merged**